Features/GoogleAnalytics/script.js

```diff
@@ -0,0 +1,35 @@
+/* global wpData, Cookies */
+
+import $ from 'jquery'
+import 'file-loader?name=vendor/js-cookie.js!js-cookie/src/js.cookie'
+
+const $gaOptoutLinks = $('.globalAction-optoutGa')
```

Comment:
I feel like a class is too restrictive for this. What if I wanted to do the opt out programmatically instead? This approach would not allow for that. Also, further down you remove all opt out "links", not giving anyone the choice to maybe use a button for example that could be set to disabled instead.

Why not just have a function available here that can be called in a component and let the component handle the visual counterpart (click event, removing the element or toggling a class)?